File: backend-springboot/src/main/java/com/javeriana/pontimovil/ponti_movil/dto/gestionar_conductores/conductor_enviado/cConductorEnviandoDTO.java
package com.javeriana.pontimovil.ponti_movil.dto.gestionar_conductores.conductor_enviado;

import com.javeriana.pontimovil.ponti_movil.dto.gestionar_conductores.conductor_recibiendo.cDireccionRecibiendoDTO;
import lombok.*;

import java.io.Serializable;
import java.util.List;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@EqualsAndHashCode
@NoArgsConstructor
public class cConductorEnviandoDTO implements Serializable {
    UUID id;
    String nombre;
    String apellido;
    String cedula;
    String telefono;
    cDireccionRecibiendoDTO direccion;
    List<String> diasSemana;
    List<cBusEnviandoDTO> buses;
}

File: frontend-angular/src/app/dto/gestionar-conductores/conductor-enviando/cConductorEnviandoDTO.ts
import {CBusEnviandoDTO} from './cBusEnviandoDTO';
import {CDireccionRecibiendoDTO} from '../conductor-recibiendo/cDireccionRecibiendoDTO';


export class CConductorEnviandoDTO {
  constructor(
    public id: string,
    public nombre: string,
    public apellido: string,
    public cedula: string,
    public telefono: string,
    public direccion: CDireccionRecibiendoDTO,
    public diasSemana: string[],
    public buses: CBusEnviandoDTO[],
  ) {}
}
